### DISCUSSION TITLE
[Feature] Chronological order slideshow

### DISCUSSION THREAD

**Discussion by jonas-jasas** [immich-app/immich#5886] [Feature Request]
### The feature

Revers slideshow direction to chronological picture order (from the past to the future).
Shuffle/No shuffle button could be used to set the direction. This button could have three state Shuffle/Future/Past.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mplonski** [comment]
👍 for this feature, I was really surprised when I found out that it's not the default behaviour!
